src/utils/requests/__tests__/BillRequests.test.js

```diff
@@ -19,7 +19,7 @@ describe("BillRequests", () => {
 
                 return new Promise((resolve) => {
                     resolve({
-                        data: {}
+                        json: () => Promise.resolve({data: {}})
```

Comment:
```suggestion
                        json: jest.fn().mockResolvedValue({data: {}})
```

https://jestjs.io/docs/en/mock-function-api#mockfnmockrejectedvaluevalue